app/shared-components/PromoModal/PromoModal.jsx

```diff
@@ -23,7 +23,7 @@ import { sendMessage } from '../../panel/utils/msg';
 import globals from '../../../src/classes/Globals';
 import ModalExitButton from '../../panel/components/BuildingBlocks/ModalExitButton';
 
-const DOMAIN = globals.DEBUG ? 'ghosterystage' : 'ghostery';
+const { GHOSTERY_ROOT_DOMAIN } = globals;
```

Comment:
you need to expand `CHECKOUT_BASE_URL` here to use it below

Reply:
@fcjr @benstrumeyer we actually should also use `GHOSTERY_ROOT_DOMAIN` / `GHOSTERY_ROOT_URL` here, as per https://github.com/ghostery/ghostery-extension/pull/594

Reply:
@wlycdgr We might as well close #594 and just update this link in this PR as well then.

Reply:
@fcjr good idea

Reply:
I missed that! Added, and also updated the insights link, as per #594